public/client-v2/src/services/capiQuery.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+import { qs } from '../util/qs';
+
+const API_BASE = 'https://content.guardianapis.com/';
+
+const capiQuery = (apiKey: string) => ({
```

Comment:
Should we return appropriate errors to users in there is something wrong with capi? What do we display here if capi is not returning any search results?

Reply:
Yeah definitely, I wasn't quite sure whether we were actually going to use CAPI (for the thing you said about preview content which I hadn't thought about) so didn't do everything there. Let's work out what out plan is there and we can definitely spec this up! 👍